docs/tech_primer.md

```diff
@@ -0,0 +1,135 @@
+# Smart-Ready Building Technical Primer
+
+This document provides  primer for smart-ready building assembly, to essentially
+answer the question *"What does Smart Ready mean?"* and *"How do we know it to be
+true?"* Specifically, it addresses the delta between a traditional site and
+smart-ready sites: places where new bleeding edge building technology is being
+used.
+
+# Key Terms & Definitions
+
+## Building Models
+
+There are different levels of "digital maturity" for different buildings. Each
+one has a slightly different scope in terms of what it means to be "complete."
+For the most part, this document addresses the scope of a Smart Ready building:
+-   **Legacy**: It works, but does not meet requirements or guidelines.
+-   **Compliant**: Meets basic security and networking requirements.
+-   **Smart Ready**: On-prem integrated with the cloud using UDMI & DBO.
+-   **Digital Building**: Completely integrated with back-end services.
+
+## Device
+
+Hardware that exists in the building that can change state in a way that can be machine communicated. A potentiometer or a flow switch is a device if it is monitored by a digital system. A commissioning valve or damper with no communication is not a device. 
+
+## Devices Classification
+
+Various adjectives qualify the different flavors of a "device" found in a building. More than one can apply, and each has a specific technical meaning/definition (i.e. they are not subjective).
+
+### Connectivity
+
+How data is moved around on-prem
+
+-   **Analog**: The device uses an analog connection (1-10V, 4-20mA etc)
```

Comment:
I think we should take out "Analog" for now.  It's not reading right, and I think you're blending the line between what is considered a "device" here.  Practically speaking, in the analog case the "device" is the thing with the ADC that's making it digital. It's not the same use of a "data moving around" as the other uses.  I'd rather keep this clean for now and add it in later if it's necessary. Unless you have a clear articulation of why it's _necessary_ to have this defined here -- but our goal is not to detail every kind of connection possible, rather the ones with salient meaning (e.g. people talk about "serial" devices all the time).

Reply:
I think it should be listed here because analog devices are a key part of building control and include things like actuators, temperature & flow sensors, etc. Although they are always connected to an ADC, which will either implement UDMI or connect over some serial interface to a gateway which does; I expect it to be common there are reporting devices which comprise only of points from multiple analog field devices, or represent a single analog field device. Therefore having the line in the tech primer makes it immediately clear that there is 'support' in UDMI for these sorts of devices.

In the industry the analog devices would be known as field devices, so the line could be  `A field device connected using an analog connection`?

Reply:
What you're describing, tho, is something fundamentally different than the other things.  You're talking about how to access a single point, while everything else is at a higher level (e.g. there's commonly the concept of an address or ID you have with serial and networked connections, but not so with analog). Practically speaking, I have never seen anybody discuss "analog" connections while doing a BOS design. Overall, UDMI doesn't expose things just because they're "industry standard" -- since that's rife with pain... rather, trying to focus on semantically what adds value to the concept.  Can you give an example of a discussion where the concept of "analog" is relevant in the same context as talking about "UDMI"?

Reply:
All points true and I cannot give an example of a discussion where it's relevant in the same context as UDMI. I've removed it